src/main/java/hudson/plugins/accurev/delegates/AbstractModeDelegate.java

```diff
@@ -267,19 +297,33 @@ protected String getStreamColorStream() {
 
     private void setStreamColor() throws IOException {
         if (isSteamColorEnabled()) {
-            //For AccuRev 6.0.x versions
-            SetProperty.setproperty(scm, accurevWorkingSpace, listener, launcher, accurevEnv, server, getStreamColorStream(), getStreamColor(), "style");
-
-            //For AccuRev 6.1.x onwards
-            SetProperty.setproperty(scm, accurevWorkingSpace, listener, launcher, accurevEnv, server, getStreamColorStream(), getStreamColor(), "streamStyle");
+        	 SetProperty.setproperty(scm, accurevWorkingSpace, listener, launcher, accurevEnv, server, getStreamColorStream(),
+                     getStreamColor(), getStreamTypeParameter());
         }
     }
-
+    /**
+     * populate the whole workspace if workspace delete option selected else populate latest transactions from the jenkins
+     * @param populateRequired
+     * @return
+     * @throws IOException
+     */
     protected boolean populate(boolean populateRequired) throws IOException {
         if (populateRequired) {
+        	 String stream = getPopulateStream();
+             int lastTransaction = NumberUtils.toInt(getLastBuildTransaction(), 0);
+             logger.info("Last transaction from jenkin " + lastTransaction);
+             String filePath = (lastTransaction == 0 || scm.isDeleteWorkspaceBeforeBuildStarts()) ? null : getFileRevisionsToBeIncluded(
```

Comment:
I would like to avoid this populate only x number of files and especially due to relying on whether prebuild cleanup is set. This will only work for certain project types. Also what if someone changed the file locally should it not overwrite the current state?